RubberduckTests/Symbols/DeclarationFinderTests.cs

```diff
@@ -455,5 +522,308 @@ private static void AddReference(Declaration toDeclaration, Declaration fromModu
         {
             toDeclaration.AddReference(toDeclaration.QualifiedName.QualifiedModuleName, fromModuleDeclaration, fromModuleDeclaration, context, toDeclaration.IdentifierName, toDeclaration, Selection.Home, new List<Rubberduck.Parsing.Annotations.IAnnotation>());
         }
+
+        #region AccessibilityTests_HardcodedModuleContent
```

Comment:
You're onto something here, I like this very much - not the `#region`, but the idea of pulling the code content - our "data" - from the tests is awesome.

I'll bring in your PR, and then let's refactor them into their own `static class` with a bunch of expression-bodied members that return a `static readonly string` - and eventually one "code" class for every test class... or hmm, one "repository" of VBA sample code to test against. @Hosch250 what do you think?